.cirrus.yml

```diff
@@ -0,0 +1,54 @@
+task:
+  trigger_type: manual
+  name: E2E/Integration Tests
+  timeout_in: 45
+  container:
+    image: ubuntu:21.04
```

Comment:
This is an awkward version now: 20.04 is LTS, and 21.10 is the newest non-LTS build.  21.04 is neither…

Reply:
Oh boy,
I thought the 21.04 was the LTS one...moving back to 20.04 for now.